pyboy/utils.py

```diff
@@ -173,7 +173,8 @@ class WindowEvent:
         DEBUG_MEMORY_SCROLL_UP,
         MOD_SHIFT_ON,
         MOD_SHIFT_OFF,
-    ) = range(41)
+        WINDOW_RESIZED,
```

Comment:
I noticed that some event IDs are prefixed with `_INTERNAL_` but wasn't sure if the resize event is also to be considered as such. Does "internal" refer to the fact that the event comes from the framework itself and shouldn't be possible to trigger from client code? If so, I would suppose window resizing counts as internal.

Reply:
Yes, internal ones, are not supposed to be called from user-code. I guess there are a few we should add `_INTERNAL` to.

Reply:
Cool! I'll change this new event to `_INTERNAL` then. (At least I cannot think of a use-case where this would be called from user-code.)